lib/roo/excelx/relationships.rb

```diff
@@ -19,6 +19,12 @@ def include_type?(type)
         end
       end
 
+      def target(type)
+        to_a.map do |_, rel|
+          rel["Target"] if rel["Type"]&.include? type
+        end.compact.first
+      end
```

Comment:
How about implementing something like following (with better method name):
```
def targets(type: "")
  return to_enum(:targets, type: type) unless block_given?

  to_a.each do |_, rel|
    yield rel["Target"] if rel["Type"]&.include? type
  end
end
```

Reply:
I was thinking there is always a single target for a type.
changed name and implemented according to your proposla.